examples/jsm/interactive/HTMLMesh.js

```diff
@@ -237,6 +237,14 @@ function html2canvas( element ) {
 
 	}
 
+	function drawImage() {
```

Comment:
I think `element`, `x`, `y`, `width`, and `height` variables should be passed here.

Reply:
Yes, I also noticed this not too long ago. I have a fix to submit soon